### DISCUSSION TITLE
Need help with plugins

### DISCUSSION THREAD

**Discussion by ghost** [JaKooLit/Fedora-Hyprland#130] [Q&A]
I'm trying to run cs2 but I'm getting black bars. I've installed csgo-vulkan-fix plugin. But still getting black bar.

**Comment by ghost** [comment]
Also where I've to add this plugin config?
```
plugin {
    csgo-vulkan-fix {
        res_w = 1680
        res_h = 1050

        # NOT a regex! This is a string and has to exactly match initial_class
        class = cs2

        # Whether to fix the mouse position. A select few apps might be wonky with this.
        fix_mouse = true
    }
}
```

**Comment by JaKooLit** [comment]
I dont use plugin in.. but from what I gather

https://wiki.hyprland.org/Plugins/Using-Plugins/
enable and download plugins required

then you can add that plugin config either on hyprland.conf
or create another file like say plugins.conf and source that on your hyprland.conf
